Tests/Packet++Test/Tests/PacketUtilsTests.cpp

```diff
@@ -1,13 +1,13 @@
 #include "../TestDefinition.h"
 #include "../Utils/TestUtils.h"
 #include "EndianPortable.h"
-#include "Packet.h"
 #include "IPv4Layer.h"
 #include "IPv6Layer.h"
+#include "Packet.h"
+#include "PacketUtils.h"
+#include "SystemUtils.h"
```

Comment:
The order of includes was changed, let's revert?

Reply:
fixed.